Failing test: APM Cypress.x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map·cy·ts - service map when navigating to service map when there is no data shows empty state shows empty state

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 60000ms: Expected to find content: 'No services available' but never did.
    at Context.eval (webpack:///./cypress/e2e/service_map/service_map.cy.ts:93:11)
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/59366#0194709b-03a6-4506-8b40-a468cb0f7793)

<!-- kibanaCiData = {"failed-test":{"test.class":"APM Cypress.x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map·cy·ts","test.name":"service map when navigating to service map when there is no data shows empty state shows empty state","test.failCount":7}} -->

--- a/x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts
+++ b/x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts
@@ -29,6 +29,7 @@ const detailedServiceMap = url.format({
   },
 });
 
+// Flaky: https://github.com/elastic/kibana/issues/207005
 describe.skip('service map', () => {
   before(() => {
     synthtrace.index(
@@ -85,7 +86,6 @@ describe.skip('service map', () => {
       );
     });
 
-    // Flaky: https://github.com/elastic/kibana/issues/207005
     describe('when there is no data', () => {
       it('shows empty state', () => {
         cy.visitKibana(serviceMapHref);

--- a/x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts
+++ b/x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts
@@ -29,7 +29,7 @@ const detailedServiceMap = url.format({
   },
 });
 
-describe('service map', () => {
+describe.skip('service map', () => {
   before(() => {
     synthtrace.index(
       opbeans({
@@ -52,7 +52,7 @@ describe('service map', () => {
       cy.intercept('GET', '/internal/apm/service-map?*').as('serviceMap');
     });
 
-    it.skip('shows nodes in service map', () => {
+    it('shows nodes in service map', () => {
       cy.visitKibana(serviceMapHref);
       cy.wait('@serviceMap');
 
@@ -68,7 +68,7 @@ describe('service map', () => {
       );
     });
 
-    it.skip('shows nodes in detailed service map', () => {
+    it('shows nodes in detailed service map', () => {
       cy.visitKibana(detailedServiceMap);
       cy.wait('@serviceMap');
       cy.contains('h1', 'opbeans-java');
@@ -85,6 +85,7 @@ describe('service map', () => {
       );
     });
 
+    // Flaky: https://github.com/elastic/kibana/issues/207005
     describe('when there is no data', () => {
       it('shows empty state', () => {
         cy.visitKibana(serviceMapHref);

